### PR TITLE
[run_experiment] Log more when we fail to create dispatcher.

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

File: experiment/run_experiment.py [hunk 1]
```diff
@@ -406,6 +405,7 @@ def start(self):
             '${WORK}/src/experiment/dispatcher.py || '
             '/bin/bash'  # Open shell if experiment fails.
         ]
+        logs.info('Starting dispatcher with container name: %s', container_name)
```
Comment (inferno-chromium) [inline experiment/run_experiment.py:408]: nit: "with instance name" to match other message

File: common/gcloud.py [hunk 2]
```diff
@@ -86,7 +87,13 @@ def create_instance(instance_name: str,
         command.extend(
             ['--metadata-from-file', 'startup-script=' + startup_script])
 
-    return new_process.execute(command, expect_zero=False, **kwargs)[0] == 0
+    result = new_process.execute(command, expect_zero=False, **kwargs)
+    if result.retcode == 0:
+        return True
+
+    logs.info('Failed to create instance. Command: %s failed. Output: %s',
```
Comment (inferno-chromium) [inline common/gcloud.py:94]: logs.error ?

**Review by inferno-chromium — APPROVED** [review]

File: experiment/run_experiment.py (hunk 1 above)
Comment (jonathanmetzman) [reply experiment/run_experiment.py:408]: It's in the command.

File: common/gcloud.py (hunk 2 above)
Comment (jonathanmetzman) [reply common/gcloud.py:94]: It's not worth reporting as an error when we are creating runners and there is no quota. So I leave it up to the caller.

**Pull request merged**